_opcodes/op114-bgee.html

```diff
@@ -0,0 +1,14 @@
+---
+n: 114
+name: "Graphics: Dither"
+param1: "Undefined"
+param2: "Undefined"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Sets <a href="../files/ids/bgee/stats.htm#61">IDENTIFYMODE</a> to 1.<br />
+Affected creature will be visible, targetable, and if controlled, selectable and orderable within the semi-transparent fog of war, but not within the black shroud.
```

Comment:
Sounds like this version needs a better name. Too bad they left it as-is.

Reply:
> Sounds like this version needs a better name.

Any suggestion? There's already a Graphics Option named like that (Dither Always), so...

Reply:
Yeah, but the ee version is about item identification (#149 set the same stat in the original), so it doesn't match. It's just confusing, we don't need to change it necessarily.